packages/exoflex/tsconfig.json

```diff
@@ -12,5 +12,5 @@
     "downlevelIteration": true,
     "types": ["react", "react-native", "jest"]
   },
-  "exclude": ["example/", "lib/"]
+  "exclude": ["example/", "new-example/", "lib/"]
```

Comment:
let's revert this since we already wipe the new-example folder

Reply:
Will do